README.md

```diff
@@ -1897,6 +1896,7 @@ Update Time, five active automations, webhooks.
 
 ## Other Free Resources
 
+  * [Smart Notes](https://smart-notes.sg-app.com) - Free cloud-based notepad with advanced features like Collaborative Editing, Nearby Sharing, Rich Text Support, and much more!
```

Comment:
looks to me its a bit early to launch being a beta, this list isnt really for beta software but usable complete products.

I accept betas in advanced stage where pricing is clear.  Feel free to come back later once you're ready.

Reply:
Thanks, will do!

I have currently kept the free forever plan open for now as it would take a while to set up the payment gateways and other stuff. It's currently in beta because of that reason.

BTW, I can create another PR to remove cyclic.sh from the list if you need it (or you can remove it directly).

Reply:
You can also update this one to remove yours easier than a new one